MediaBrowser.Common/Net/HttpRequestOptions.cs

```diff
@@ -64,12 +64,6 @@ public string Host
             set => RequestHeaders[HeaderNames.Host] = value;
         }
 
-        /// <summary>
-        /// Gets or sets the progress.
-        /// </summary>
-        /// <value>The progress.</value>
-        public IProgress<double> Progress { get; set; }
```

Comment:
Why remove it?

Reply:
Because it isn’t used anymore in the HttpClientManager